lib/discourse_chat/provider/slack/slack_message.rb

```diff
@@ -49,8 +49,22 @@ def text
       text
     end
 
+    def processed_text_with_attachments
+      string = self.text
+      string += "\n" if !attachments.empty?
+      attachments.each do |attachment|
+        string += " - #{attachment}\n"
+      end
+      string
+    end
+
     def raw_text
-      @raw['text']
+      string = @raw['text']
+      string += "\n" if !attachments.empty?
+      attachments.each do |attachment|
+        string += " - #{attachment}\n"
+      end
+      string
```

Comment:
We're duplicating logic in the two methods. Perhaps we can extract the common into a method somehow?